README Example should indicate actions/checkout@master is required

<!-- Thank you for reporting a possible bug.  Please fill in as much of the template below as you can. -->

**Describe the bug**

The Example in `README.md` doesn't make it clear that `-uses: actions/checkout@master` is required.  The comment "# Remove or modify this step as needed" introduces further confusion.

**Steps to Reproduce**
1. Add a file in `.github/workflows/wordpress-plugin-deploy.yml`
```
name: Release pantheon-hud plugin to wp.org
on:
  release:
    types: [published]
jobs:
  release:
    runs-on: ubuntu-latest
    steps:
    - name: WordPress Plugin Deploy
      uses: 10up/action-wordpress-plugin-deploy@2.1.1
      env:
        SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
        SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
        SLUG: ${{ env.SLUG }}
```
2.  See this error when publishing a release:
```
> Copying files...
ℹ︎ Using .gitattributes
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
tar: This does not look like a tar archive
tar: Exiting with failure status due to previous errors
Error: Process completed with exit code 2.
```

**Expected behavior**

The action would deploy the code in the release to WordPress' SVN server.

**Additional context**

While debugging the action, we found these two comments on issues where others had done similar things.
* https://github.com/10up/action-wordpress-plugin-deploy/issues/11#issuecomment-546536900
> You’ve commented out the - uses: actions/checkout@master step, which is necessary to have any of the GitHub repo available for use during the run. 
* https://github.com/10up/action-wordpress-plugin-deploy/issues/32#issuecomment-609786150
> You're missing the line - uses: actions/checkout@master in your tag deploy workflow :) Without that, it will not have any of the contents you're expecting, and it will not be a Git checkout.

--- a/README.md
+++ b/README.md
@@ -80,7 +80,7 @@ Current set of example workflow files:
 
 **Note**: The following step is required to check out the repository for use during the workflow run:
 ```
-- uses: actions/checkout@main
+- uses: actions/checkout@v4
 ```
 
 * [Deploy on pushing a new tag and create release with attached ZIP](examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml)

--- a/README.md
+++ b/README.md
@@ -77,6 +77,12 @@ Current set of example workflow files:
 
 * [Deploy on publishing a new release and attach a ZIP file to the release](examples/deploy-on-publishing-a-new-release-and-attach-a-zip-file-to-the-release.yml)
 * [Deploy on pushing a new tag](examples/deploy-on-pushing-a-new-tag.yml)
+
+**Note**: The following step is required to check out the repository for use during the workflow run:
+```
+- uses: actions/checkout@main
+```
+
 * [Deploy on pushing a new tag and create release with attached ZIP](examples/deploy-on-pushing-a-new-tag-and-create-release-with-attached-zip.yml)
 
 ## Contributing

